chronos_bin/src/telemetry/otlp_collector.rs

```diff
@@ -0,0 +1,40 @@
+use opentelemetry::trace::TraceError;
+use opentelemetry::{
+    global,
+    sdk::{propagation::TraceContextPropagator, trace as sdktrace},
+};
+use opentelemetry_otlp::{Protocol, WithExportConfig};
+
+pub struct OtlpCollector {}
+
+impl Default for OtlpCollector {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl OtlpCollector {
+    pub fn new() -> Self {
+        OtlpCollector {}
+    }
+
+    pub fn http_collector_connect(&self, protocol: Protocol) -> Result<sdktrace::Tracer, TraceError> {
```

Comment:
@fmarek-kindred  this is a method which is calling the http protocol as per the input